src/app/shared/components/managecitations/managecitations.component.scss

```diff
@@ -2,10 +2,37 @@
     padding-top: 15px;
 }
 
-
 button {
     margin: 0px 5px 5px 0px;
 }
 .modal-body{
     overflow-y: auto;
 }
+
+.regRegion {
+    width: 228px; 
+    border-top-style:none; 
+    border-left-style:none; 
```

Comment:
These could be moved to a common css styling since they're used in all of them - this can be done by setting the class for the div that all the regRegion/editRegRegion, etc. are in. Then, say you title it `class="regRegionDiv"` or something like that, you can then set the styling you want all the tds inside it to have by using:
```
.regRegionDiv td {
    border-top-style: none;
    border-left-style: none;
}
```

Reply:
Oh, that's a good idea! I was thinking each class was slightly different, but it's true most of them are nearly the same with one attribute added/removed.